EXTENSION.md

```diff
@@ -2,39 +2,150 @@
 
-- Code completion
-- Go to definition, type definition and references
-- Symbol renames
-- Solidity code formatting through [prettier-plugin-solidity](https://github.com/prettier-solidity/prettier-plugin-solidity)
-- Inline code validation from compiler errors/warnings for Hardhat projects
-- Hover help for variables, function calls, errors, events etc.
-- Code actions (quickfixes) suggested from compiler errors/warnings for Hardhat projects
-  - Implement missing functions on interface with stubs
-  - Constrain mutability by adding `view`/`pure` to function signature
-  - Meet inheritance requirements by adding `virtual`/`override` on function signature
-  - Provide accessibility by adding `public`/`private` to function signature
+- [Code completion](#code-completions)
+- [Go to definition, type definition and references](#navigation)
+- [Symbol renames](#renames)
+- [Solidity code formatting](#format-document)
+- [Inline code validation from compiler errors/warnings for Hardhat projects](#inline-code-validation-diagnostics)
+- [Hover help for variables, function calls, errors, events etc.](#hover)
+- [Code actions (quickfixes) suggested from compiler errors/warnings for Hardhat projects](#code-actions)
+  - [Implement missing functions on interface with stubs](#implement-missing-functions-on-interface)
+  - [Constrain mutability by adding `view`/`pure` to function signature](#constrain-mutability)
+  - [Meet inheritance requirements by adding `virtual`/`override` on function signature](#adding-virtualoverride-on-inherited-function-signature)
+  - [Provide accessibility by adding `public`/`private` to function signature](#adding-publicprivate-to-function-signature)
 
 Built by the [Nomic Foundation](https://nomic.foundation/). [We’re hiring](https://nomic.foundation/hiring).
 
-<hr />
+---
 
 - [Hardhat for Visual Studio Code](#hardhat-for-visual-studio-code)
   - [Installation](#installation)
+  - [Features](#features)
   - [Setup](#setup)
   - [Hardhat Projects](#hardhat-projects)
     - [Monorepo Support](#monorepo-support)
   - [Formatting](#formatting)
     - [Formatting Configuration](#formatting-configuration)
   - [Feedback, help and news](#feedback-help-and-news)
 
-<hr />
+---
 
 ## Installation
 
 **Hardhat for Visual Studio Code** can be installed by [using the Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity).
 
 Some features (e.g. inline validation, quick fixes) are still experimental and are only enabled within a [Hardhat](https://hardhat.org/) project, this is a limitation that will be lifted with future releases.
 
+## Features
+
+### Code Completions
+
+In addition to reference completion (e.g. contract instances, globally available variables and built-in types like arrays), **Hardhat for Visual Studio Code** has completions on import directives.
```

Comment:
I find this a bit difficult to understand. What about something like:

```suggestion
**Hardhat for Visual Studio Code** autocomplete references to existing symbols (e.g. contract instances, globally available variables and built-in types like arrays) and import directives (i.e. it autocompletes the path to the imported file).
```

Feel free to change the wording, my suggestion is mostly about ordering.